pkg/ipamd/rpc_handler.go

```diff
@@ -203,9 +203,9 @@ func (s *server) AddNetwork(ctx context.Context, in *rpc.AddNetworkRequest) (*rp
 
 	if s.ipamContext.enablePodIPAnnotation {
```

Comment:
Right, we can skip for v4 and v6 nil address..since this fix was done for calico I would recommend to check if calico expects nil ip for the label or no label at all...